src/project/views/NewUserView.java

```diff
@@ -46,18 +50,24 @@ public static void newUserView(Stage stage){
 		provinceBox.add("QC");
 		provinceBox.add("SK");
 		provinceBox.add("YT");
+		String province = provinceCombo.getValue().toString();
 		Label postCodeLabel = new Label("Postal Code:");
 		postCodeField = new TextField();
+		String postalCode = postCodeField.getText();

```

Comment:
Address should be passed as a single string, we don't use it for much so I think just create a string maybe comma separated or something would be fine.